docs/releases/1.10.txt

```diff
@@ -248,6 +248,8 @@ Internationalization
   used in a root URLConf specified using :attr:`request.urlconf
   <django.http.HttpRequest.urlconf>`.
 
+* New parameter `prefix_default_language` for :func:`~django.conf.urls.i18n.i18n_patterns`.
```

Comment:
It's useful to describe what the parameter does, e.g. "The new ``prefix_default_language`` parameter for :func:`~django.conf.urls.i18n.i18n_patterns` allows..." (also wrap lines at 79 characters)